xml_inscriptions/metadata_only/NJ.Princ.PU.AM.G.c399-I310.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- v. 20. 2019-10-04 EM  -->
+<!--<?oxygen RNGSchema="http://cds.library.brown.edu/projects/usepigraphy/schema/exp-epidoc.rng" type="xml"?>
+--><TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:xi="http://www.w3.org/2001/XInclude" >
+    <teiHeader>
+
+        <fileDesc>
+
+            <titleStmt>
+                <title>NJ.Princ.PU.AM.G.c399-I310</title>
+            </titleStmt>
+
+            <publicationStmt>
+                <authority>Brown University</authority>
+                <idno type="USEpigraphy" xml:id="NJ.Princ.PU.AM.G.c399-I310">Repeat USEP ID</idno>
+            </publicationStmt>
+
+            <sourceDesc>
+                <!--  *************************************  <msDesc>  ********************************* -->
+                <msDesc>
+                    <msIdentifier>
+                        <region>NJ</region>
+                        <settlement>Princ</settlement>
+                        <institution>PU</institution>
+                        <repository>AM</repository>
+                        <idno>c399-I310</idno>
+                    </msIdentifier>
+
+                    <!--  *************************************  <msContents>  *********************************    -->
+                    <msContents>
+                        <textLang mainLang="grc" otherLangs=""/>
+                        <!-- arc, grc, he, lat, und (unknown),  zxx (non-linguistic content) -->
+                        <!-- could it be latin? -->
+                        <msItem>
```

Comment:
Many of the smaller fragments aren't given any classification, so should I tag those as msItem class="#text_unknown" rather than leave the msItem blank?

Reply:
I think that that makes the most sense; if nothing else, you could leave it blank but leave the `@class` there rather than taking it out, since it means that it might not pull in a search.